.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## Description of changes:
+
+## CTSM Issues Fixed (include github issue #):
+(https://github.com/ESCOMP/ctsm/issues)
```

Comment:
I don't think the link to issues is necessary here. If someone has fixed an issue, they presumably know where to find issues.